src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -776,66 +776,5 @@ internal BoundStatement WrapWithVariablesAndLocalFunctionsIfAny(CSharpSyntaxNode
                                   ImmutableArray.Create(statement))
             { WasCompilerGenerated = true };
         }
-
-        internal string Dump()
```

Comment:
No.  We still run things in the debugger when in release mode.  Now, maybe we should introduce an attribute to call out that this is intentional and is not dead code.  Then your tool could skip those guys.